cli/parse-options.js

```diff
@@ -18,6 +19,9 @@ var log = require('npmlog')
  * passe set the `inMemory: true` option. If it’s not set, leveldown is used
  * with the prefix set to `options.data` + 'data' (`.hoodie/data` by default).
  */
+
+PouchDB.plugin(require('pouchdb-mapreduce'))
+
```

Comment:
yes, exactly! Remember this pull request is about removing `options.db`, and lines 31-58 in `server/index.js` are handling that option right now. So instead we have to move all the handling of it before we pass options to `server/index.js`, and `cli/parse-options.js` is the right place for it